fossology/FOSSologyUI
# added storybook for button

## Description

Added Storybook for button widget

### Changes

Added Storybook for button and deleted default storybook items

## How to test

run npm run storybook and verify that Button storybook is displayed

Please consider using the closing keyword if the pull request is proposed to
fix an issue already created in the repository
(https://help.github.com/articles/closing-issues-using-keywords/)
